docs/state.md

```diff
@@ -435,8 +435,14 @@ WindowStore
       }
     },
     hasBitcoinHandler: boolean,  // brave browser has a `bitcoin:` URI handler
-    paymentIMG: string,           // the QR code equivalent of `paymentURL` expressed as "data:image/...;base64,..."
-    error: { // non-null if the last updateLedgerInfo happened concurrently with an error
+    countryCode:                 // ISO3166 2-letter code for country of browser's location
+    exchangeInfo: {              // information about corresponding "friendliest" BTC exchange (suggestions welcome!)
+      titleText: string,         // e.g., 'Create an account on ...'
+      buttonText: string,        // e.g., 'Create'
+      buttonURL: string,         // where the button should take the browser      
```

Comment:
i suspect that localization has to occur in the browser in the ui subsystem. 

is there really a better key than just a text string? both `buttonText` and `titleText` can be used as the key values. am i missing something?